app/views/includes/no-results.nunjucks

```diff
@@ -5,8 +5,15 @@
     <div class="results-block results-block--none">
       <h2 class="results__header results__header--none">{% block resultsHeader %}{% endblock %}</h2>
       <div class="results__none-content">
-        <p>This service only provides information about pharmacies in England.</p>
-        {% block reasonForMessage %}<p>You might be getting this result because you live in Scotland, Wales or Northern Ireland, or because you live near a border with one of those countries. It could also be because you live on the Isle of Man.</p>{% endblock %}
+        {% block reasonForMessage %}
+        <p>If the place you searched for is in England, you could:
```

Comment:
This message is show for postcode searches which doesn't marry up to well with the reference to
> the place you search for...

Or the bullets talking about typos and place name names.

https://connecting-to-services-pr-393.dev.beta.nhschoices.net/find-a-pharmacy/results?location=g1